torch/csrc/jit/passes/onnx/constant_fold.cpp

```diff
@@ -265,6 +268,34 @@ c10::optional<at::Tensor> runTorchBackendForOnnx(
       }
     }
     return c10::optional<at::Tensor>(at::reshape(updated_val, shape));
+  } else if (node->kind() == onnx::ReduceL1 || node->kind() == onnx::ReduceL2) {
+    assert(inputTensorValues.size() == 1);
```

Comment:
nit : this function is starting to be pretty long. we should consider moving the logic inside this 'if' clauses to separate functions for readability